micro-ui/web/micro-ui-internals/packages/modules/core/CHANGELOG.md

```
@@ -1,26 +1,31 @@
 ## [1.8.38] - [06-June-2025]
-- Added `Fragment` wherever required as older version of react doesn't support <></>
-- PrivateRoute is using `component   as a prop in index.js file
+- Added Fragment wherever required as older version of react doesn't support <></>
+  PrivateRoute is using `component as a prop in index.js file
 
 ## [1.8.37] - [04-June-2025]
-- Added an additional component `Carousel.js`
-- config updated as per additional carousel component
-- Updated `FormComposer` usage in `login.js` with conditional banner rendering.
-- Leveraged newly added class `.login-form-container` for styling the Login page.
+- Added an additional component Carousel.js
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)